command/tls/ca/create/tls_ca_create_test.go

```diff
@@ -120,6 +121,14 @@ func expectFiles(t *testing.T, caPath, keyPath string) (*x509.Certificate, crypt
 	require.FileExists(t, caPath)
 	require.FileExists(t, keyPath)
 
+	fi, err := os.Stat(keyPath)
+	if err != nil {
+		t.Fatal("should not happen", err)
+	}
+	if want, have := fs.FileMode(0600), fi.Mode().Perm(); want != have {
+		t.Fatalf("private key file %s: permissions: want: %o; have: %o", keyPath, want, have)
+	}
```

Comment:
I generally we try to use `require.Equal` to match the other assertions, but this is ok too.

Reply:
Hello @dnephin, I had noticed that consul uses testify; I went with this style is because I am used to it and thought that I could not express the same with testify, but now that I think about it, I could have passed the two integers to require.Assert; on the other hand I wanted to put in the error message also the file path... I should have searched before if there was a way to do so with testify.